README.md

```diff
@@ -20,19 +22,28 @@ npm install mocha-lcov-reporter --save-dev
 
 ## Usage:
 
-This script ( `bin/coveralls.js` ) can take standard input from any tool that emits the lcov data format (including [mocha](http://mochajs.org/)'s [LCov reporter](https://npmjs.org/package/mocha-lcov-reporter)) and send it to coveralls.io to report your code coverage there.
+This script can take standard input from any tool that emits the lcov data format (including [mocha](http://mochajs.org/)'s [LCov reporter](https://npmjs.org/package/mocha-lcov-reporter)) and send it to coveralls.io to report your code coverage there.
+
+Once your app is instrumented for coverage, and building, you need to pipe the lcov output to `npx coveralls`.  In order to communicate coverage results on your behalf, node-coveralls needs to know the secret coveralls repo token for your Coveralls.io repo.  You will, therefore, need to set COVERALLS_REPO_TOKEN in the environment: 
+```sh
+COVERALLS_REPO_TOKEN=****** npx coveralls
+```
+
+### Optional Settings
+There are optional environment variables for controlling information sent to Coveralls.  Many of these are set automatically, but can be overridden:
+* COVERALLS_PARALLEL: indicates that multiple coverage reports will be submitted to Coveralls (more info here: https://github.com/nickmerwin/node-coveralls/issues/134)
+* COVERALLS_SERVICE_NAME:  the name of your build system
+* COVERALLS_SERVICE_NUMBER: (used in conjunction with COVERALLS_PARALLEL) a unique ID used to combine coverage from several jobs into a single coverage report
```

Comment:
Service number is useful even when not used in conjunction with COVERALLS_PARALLEL, should be unique and the same for all parallel jobs, but also useful to have these ids as they can often be used to link back to builds from coveralls

Reply:
Is there a better name for that variable, then?  The `_PARALLEL` suffix would seem to imply that it's a toggle specific to parallel builds.